jwst/datamodels/container.py

```diff
@@ -64,11 +69,12 @@ class ModelContainer(model_base.DataModel):
     # does not describe the data contents of the container.
     schema_url = "container.schema"
 
-    def __init__(self, init=None, **kwargs):
+    def __init__(self, init=None, allowed_exptypes=None, **kwargs):
```

Comment:
This extra `allowed_exptypes` parameter should be added to the "Parameters" docstring above.

Reply:
Do you think `exptype` instead of `allowed_exptypes` would be more succinct?  Or `asn_exptype` to be explicit that it is an association exptype and not a FITS keyword EXPTYPE?

Reply:
I went with asn_exptype to be explicit.